File: Pemrograman1/Modul5/C/PRAK503-2310817310009-MuhammadFauzanAhsani.c
#include <stdio.h>

int maksimal(int a, int b){
    if (a < b) {
        return b;
    }
    else{
        return a;
    }
}

int minimal(int a, int b){
    if (a > b){
        return b;
    }
    else{
        return a;
    }
}

int main(){
    int batas = 0;
    int maks = -100000;
    int minim = 100000;
    int bilangan;

    scanf("%d", &bilangan);

    while (batas < bilangan){
        int nilai;
        scanf("%d", &nilai);
        maks = maksimal(maks, nilai);
        minim = minimal(minim, nilai);
        batas++;
    }
    printf("%d %d", maks, minim);
    return 0;
}

File: Pemrograman1/Modul5/Python/PRAK503-2310817310009-MuhammadFauzanAhsani.py
def maksimal(a:int, b:int):
    if (a > b):
        return a
    else:
        return b

def minimal(a:int, b:int):
    if (a < b):
        return a
    else:
        return b

batas = 0
maks = -100000
minim = 100000
bilangan = int(input())
nilai = list(map(int, input().split()))

while batas < bilangan:
    maks = maksimal(maks, nilai[batas])
    minim = minimal(minim, nilai[batas])
    batas += 1

print(f"{maks} {minim}")